customer-satisfaction/pipelines/training_pipeline.py

```diff
@@ -1,23 +1,29 @@
-from zenml.config import DockerSettings
-from zenml.integrations.constants import MLFLOW
-from zenml.pipelines import pipeline
+from typing import Tuple, Annotated
 
-docker_settings = DockerSettings(required_integrations=[MLFLOW])
+from sklearn.base import RegressorMixin
+from zenml import pipeline
 
+from pipelines.utils import docker_settings, model_version
+from steps import ingest_data, clean_data, train_model, evaluation, \
+    model_promoter
+
+
+@pipeline(
+    enable_cache=True,
+    settings={"docker": docker_settings},
+    model_version=model_version
+)
+def customer_satisfaction_training_pipeline(
+    model_type: str = "lightgbm"
+) -> Tuple[Annotated[RegressorMixin, "model"], Annotated[bool, "is_promoted"]]:

```

Comment:
no need to type annotate pipeline outputs this way

Reply:
so i shouldn't type annotate at all or just not using "Annotated"

Reply:
Don’t annotate at all